libs/perun/components/src/lib/members-dynamic-list/members-dynamic-list.component.ts

```diff
@@ -193,6 +213,8 @@ export class MembersDynamicListComponent implements AfterViewInit, OnInit, OnCha
   }
 
   changeExpiration(member: RichMember, groupId?: number): void {
+    // Skip crating the status dialog, if the member is indirect
+    if (isMemberIndirect(member)) return;
```

Comment:
please delete these lines, they are not useful, because this method is called only from changeStatus method from which this checks are already done